minichain/pow.py

```
@@ -23,20 +23,20 @@ def mine_block(
     """Mines a block using Proof-of-Work without mutating input block until success."""
     max_nonce = max_nonce if max_nonce is not None else MINING_MAX_NONCE
 
-    difficulty = difficulty if difficulty is not None else block.difficulty
-    if not isinstance(difficulty, int) or difficulty <= 0:
-        raise ValueError("Difficulty must be a positive integer.")
+    target = target if target is not None else block.target
+    if not isinstance(target, int) or target <= 0:
+        raise ValueError("Target must be a positive integer.")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)